requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+pdbpp>=0.10.2
+pytest>=6.2.1
+pypandoc>=1.5
+pytest-cov>=2.10.1
+coveralls>=2.2.0
```

Comment:
>pdbpp

Actually, it will be installed only in dev mode, so it's not necessary to be so strict about it.
Developers will install something like that for debugging anyway.

> pypandoc
But it's not extra you use it in setup.py

>no new version restrictions for pytest and others
OK, I'll leave it without restrictions at all.

>(dataclasses should be here).
You are wrong. It's not a dev dependency it should be listed in requirements.txt

Reply:
Ok, so point by point:
* `pdbpp` – it's not being installed now and it shouldn't randomly appear here. It's not related to this project at all and if someone wants it for whatever reason (I don't know what does it even do, never used it) they're free to do that.
* `pypandoc` – it's optional, not installed anywhere except for the machine of a person doing a release, shouldn't appear randomly in a change that's mostly about moving around *existing* depenendcies
* `dataclasses` - it's not a dependency, `Injector` doesn't use it or depend on it, it shouldn't randomly start appearing in dependencies list. It's purely a development dependency, it's only used in the test suite.

Reply:
>pdbpp

OK.

>pypandoc

OK, then we just need to have `release` in extras_require with this package.
Just not to mention this package at all is not correct.

> dataclasses

Yes, it is. I just thought it's a dependency cause you put it into `install_requires`

Reply:
It's not in `install_requires`.

Reply:
>It's not in install_requires.

Hm, yeah, you are right, my bad, sorry.